docs/configuring-ingest.asciidoc

```diff
@@ -56,20 +55,19 @@ For example, consider the following pipeline in a file named `pipeline.json`:
 }
 ------------------------------------------------------------------------------
 
-To register the pipeline run:
+To register this pipeline, run:
 
 [source,shell]
 ------------------------------------------------------------------------------
 curl -H 'Content-Type: application/json' -XPUT 'http://localhost:9200/_ingest/pipeline/test-pipeline' -d @pipeline.json
```

Comment:
maybe not in this change, but what do you think of changing all of the elasticsearch sample endpoints to `http://elasticsearch:9200` ?

Reply:
At some point, yes. Beats still uses localhost > elasticsearch in most places, so I think it makes sense for us to wait until more of the shared docs use elasticsearch.